.semaphore/semaphore.yml

```diff
@@ -47,21 +59,24 @@ blocks:
         type: e1-standard-4
         os_image: ubuntu1804
     jobs:
-    - name: Build
+    - name: Build and run UT, k8sfv
```

Comment:
Does this name change mean that occurrences of `dependencies: ["Build"]` need to be updated?

Reply:
No, the dependency is based on the name of the block, not the individual job.